python/requirements.txt

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-attrs==21.4.0
```

Comment:
update: i was confused and we still use `attrs` through the old `attr` api.
the removal just happened to not break things because `attrs` is a transitive dependency of `desert`, but we should not rely on that, since we use the library in our code directly.